### PR TITLE
MAUI TreeView gettig started ug

### PR DESCRIPTION
**Pull request by Lakshminatarajan**
Added Overview and Getting started UG.

### CODE REVIEW

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review

**Review by M-A32 — APPROVED** [review]

File: MAUI/TreeView/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,860 @@
...
+
+# Getting Started with .NET MAUI TreeView (SfTreeView)
+
+This section provides a quick overview for getting started with the TreeView with the .NET Maui TreeView(SfTreeView) for Maui. Walk through the entire process of creating real world of this control.
+
+
+## Creating an application using the .NET MAUI TreeView
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. The Syncfusion .NET MAUI components are available on nuget.org. To add SfTreeView to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.TreeView and then install it.
```
Comment (PiruthivirajM) [inline MAUI/TreeView/getting-started.md:18]: add link for "nuget.org" (refer our other controls UG)

File: MAUI/TreeView/overview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+documentation: ug
+---
+
+# Overview of .NET MAUI TreeView (SfTreeView)
+
+The Syncfusion .NET MAUI TreeView is a data-oriented control that displays data in a hierarchical structure with expanding and collapsing nodes. It is commonly used to illustrate a folder structure, or nested relationships in an application.
+
+## Key features
+
+ * Optimized view reusing strategy for enhanced performance.
```
Comment (PiruthivirajM) [inline MAUI/TreeView/overview.md:16]: replace with below content for Key features.

* Enhanced performance: Uses an optimized view-reusing strategy.
* Bound and unbound modes: Bind hierarchical data or manually add unbound tree nodes.
* Selection: Select nodes through different selection modes and keyboard navigation.
* Item templating: Customize the appearance of each node.
* Expand and collapse nodes: Easily reveal or hide underlying information.

File: MAUI/TreeView/overview.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+title: About .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about introduction of the Syncfusion .NET MAUI TreeView (SfTreeView) control, its elements and more.
+platform: MAUI
+control: TreeView
+documentation: ug
+---
+
+# Overview of .NET MAUI TreeView (SfTreeView)
+
+The Syncfusion .NET MAUI TreeView is a data-oriented control that displays data in a hierarchical structure with expanding and collapsing nodes. It is commonly used to illustrate a folder structure, or nested relationships in an application.
```
Comment (PiruthivirajM) [inline MAUI/TreeView/overview.md:12]: The Syncfusion .NET MAUI TreeView is a data-oriented control that displays the hierarchical data,such as organizational structures and nested connections, within your application. It provides a user-friendly way to interact with complex data structures and navigate through different levels of information by allowing users to expand and collapse nodes to reveal or hide underlying information.

**Review by PiruthivirajM — COMMENTED** [review]
Please address the review comments.

File: MAUI/TreeView/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,843 @@
+---
+layout: post
+title: Getting Started with .NET MAUI TreeView control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Changed the title as mentioned above.

File: MAUI/TreeView/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,843 @@
...
+
+# Getting Started with .NET Maui TreeView (SfTreeView)
+
+This section provides a quick overview for getting started with the TreeView with the .NET Maui TreeView(SfTreeView) for Maui. Walk through the entire process of creating real world of this control.
+
+
+## Creating an application using the .NET MAUI TreeView
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfTreeView to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.TreeView and then install it.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Changed the content as mentioned above.

File: MAUI/TreeView/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,843 @@
...
+        {
+            PropertyChanged(this, new PropertyChangedEventArgs(_PropertyName));
+        }
+    }
+}
+ 
+{% endhighlight %}
+{% endtabs %}
+
+N> If you need your hierarchical data model to respond to property changes, then implement the `INotifyPropertyChanged` interface in your model class.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Changed the content as mentioned above.

File: MAUI/TreeView/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,843 @@
...
+        return nodeImageInfo;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Bind to a Hierarchy Property Descriptors data source - Bound mode
+
+You can create a tree view by binding the ItemsSource to a hierarchy property descriptors data source. To create a tree view using hierarchical data binding, set a hierarchical collection to the `ItemsSource` property, and then set the `TargetType` and ChildPropertyName property values in HierarchyPropertyDescriptors .
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Resolved the conversation as mentioned above

File: MAUI/TreeView/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,843 @@
...
+{% endtabs %}
+
+
+## Interacting with a tree view
+
+The TreeView allows you to expand and collapse the nodes either by user interaction on the nodes or by programmatically. The expanding and collapsing interactions can be handled with the help of NodeCollapsing and NodeExpanding events.
+
+You can define how the nodes to be expanded while loading the TreeView by using AutoExpandMode property. Also, the TreeView allows you to set the restrictions whether expanding and collapsing of nodes can be performed only by tapping in expander view or in both expander view and content view by using ExpandActionTarget property.
+
+N> `AutoExpandMode` property is only applicable for bound mode. For Unbound mode you need to set `IsExpanded` property to `true` while creating the nodes, to be in expanded state while loading the TreeView.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Resolved the conversation as mentioned above

File: MAUI/TreeView/overview.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: About .NET MAUI TreeView control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI TreeView (SfTreeView) control, its elements and more.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/overview.md]: Changed the description for overview as given above

File: MAUI/TreeView/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,843 @@
...
+        MainPage = new ContentPage { Content = treeView};
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Changed the content as mentioned above

File: MAUI/TreeView/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,843 @@
...
+        }
+    }
+}
+ 
+{% endhighlight %}
+{% endtabs %}
+
+N> If you need your hierarchical data model to respond to property changes, then implement the `INotifyPropertyChanged` interface in your model class.
+
+Create a model repository class with ImageNodeInfo collection property initialized with required number of data objects in a new class file as shown in the following code example, and save it as FileManagerViewModel.cs file:
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: changed the content as mentioned above

File: MAUI/TreeView/overview.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: About .NET MAUI TreeView control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/overview.md]: Changed the title for overview as given above

File: MAUI/TreeView/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,843 @@
...
+SfTreeView treeView = new SfTreeView();
+FileManagerViewModel viewModel = new FileManagerViewModel ();
+treeView.ItemsSource = viewModel.ImageNodeInfo; 
+MainPage = new ContentPage { Content = treeView };
+{% endhighlight %}
+{% endtabs %}
+
+## Defining a template to expander and content view
+
+By defining the ExpanderTemplate and ItemTemplate properties, a custom user interface (UI) can be created to display the data items for both expander and content view. It is applicable for both Unbound mode and Bound mode data items.
```
[the remark this answers is not in the document]
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Changed the content as mentioned above

File: MAUI/TreeView/getting-started.md (hunk 1 above)
Comment (Dhivya-SF4094) [reply MAUI/TreeView/getting-started.md]: Added link for nuget.org and highlighted the package name

File: MAUI/TreeView/overview.md (hunk 2 above)
Comment (Dhivya-SF4094) [reply MAUI/TreeView/overview.md]: Replaced the key features.

File: MAUI/TreeView/overview.md (hunk 3 above)
Comment (Dhivya-SF4094) [reply MAUI/TreeView/overview.md]: Replaced with the above content

**Pull request closed without merging**